tests/lib/test_phase_stats.py

```diff
@@ -278,8 +279,126 @@ def test_phase_stats_single_network_procfs():
     assert data[14]['sampling_rate_95p'] == 100477, '95p sampling rate not in expected range'
     assert isinstance(data[14]['sampling_rate_95p'], int)
 
-
-def test_sci():
+def test_phase_stats_network_data():
+    run_id = Tests.insert_run()
+    Tests.import_network_io_cgroup_container(run_id)
+
+    test_sci_config = {
+        'N': 0.001,    # Network energy intensity (kWh/GB)
+        'I': 500,      # Carbon intensity (gCO2e/kWh)
+    }
+
+    build_and_store_phase_stats(run_id, sci=test_sci_config)
+
+    # Network energy data
+    network_energy_data = DB().fetch_all(
+        'SELECT metric, detail_name, unit, value, type, phase FROM phase_stats WHERE phase = %s AND metric = %s',
+        params=('004_[RUNTIME]', 'network_energy_formula_global'), fetch_mode='dict'
+    )
+
+    assert len(network_energy_data) == 1, f"Expected 1 network energy formula entry, got {len(network_energy_data)}"
+
+    network_energy_entry = network_energy_data[0]
+    assert network_energy_entry['metric'] == 'network_energy_formula_global'
+    assert network_energy_entry['detail_name'] == '[FORMULA]'
+    assert network_energy_entry['unit'] == 'uJ'
+    assert network_energy_entry['type'] == 'TOTAL'
+    assert network_energy_entry['phase'] == '004_[RUNTIME]'
+
+    network_totals = DB().fetch_all(
+        'SELECT detail_name, value FROM phase_stats WHERE phase = %s AND metric = %s',
+        params=('004_[RUNTIME]', 'network_total_cgroup_container'), fetch_mode='dict'
+    )
+    total_network_bytes = sum(row['value'] for row in network_totals)
+    expected_network_energy_kwh = Decimal(total_network_bytes) / 1_000_000_000 * Decimal(test_sci_config['N'])
+    expected_network_energy_uj = expected_network_energy_kwh * 3_600_000_000_000
+    assert abs(network_energy_entry['value'] - expected_network_energy_uj) < 1, \
```

Comment:
can you please use `math.isclose()` here for consistency. See example in `test_metric_importer.py`